Makefile

```diff
@@ -261,7 +272,7 @@ TEST_TARGET ?= test
 
 test-containerized:
 	$(DOCKER) run \
-	-it -e container=yes \
+	-e container=yes \
```

Comment:
IMHO `container`=`yes` meaning use of `-buildvcs=false` for `go test` could be clearer.  

Variable could be e.g. `TEST_NO_BUILDVSC`.